powerauth-restful-security-spring-annotation/src/main/java/io/getlime/security/powerauth/rest/api/spring/annotation/support/PowerAuthEncryptionArgumentResolver.java

```diff
@@ -81,11 +80,11 @@ public Object resolveArgument(@NonNull MethodParameter parameter, ModelAndViewCo
             }
         }
         // Ecies encryption object is inserted into parameter which is of type PowerAuthEciesEncryption
-        if (eciesObject != null && EciesEncryptionContext.class.isAssignableFrom(parameter.getParameterType())) {
+        if (eciesObject != null && EncryptionContext.class.isAssignableFrom(parameter.getParameterType())) {
             // Set ECIES scope in case it is specified by the @PowerAuthEncryption annotation
-            PowerAuthEncryption powerAuthEncryption = parameter.getMethodAnnotation(PowerAuthEncryption.class);
+            io.getlime.security.powerauth.rest.api.spring.annotation.PowerAuthEncryption powerAuthEncryption = parameter.getMethodAnnotation(io.getlime.security.powerauth.rest.api.spring.annotation.PowerAuthEncryption.class);
```

Comment:
Why is the FQN for package required? It can be handled by an import.

Reply:
Replaced by import.